src/server/eventcmds.cpp

```diff
@@ -755,6 +755,37 @@ DevVarLongStringArray *DServer::zmq_event_subscription_change(const Tango::DevVa
         action = (*argin)[2];
         event = (*argin)[3];
 
+//
+// Check event type validity
```

Comment:
Whenever there is a comment in the code describing what it is doing - extract a method.